packages/documentation/src/content/docs/integration/deployment.md

```diff
@@ -61,6 +61,7 @@ Now, the Admin UI can be found on localhost:3010.
 | `CONNECTOR_PORT`                                      | backend.port.connector                                                                                                                  | `3002`                                                      | port of the ILP connector for for sending packets over ILP over HTTP                                                                                                                                       |
 | `DATABASE_URL`                                        | backend.postgresql.host, backend.postgresql.port, backend.postgresql.username, backend.postgresql.database, backend.postgresql.password | `postgresql://postgres:password@localhost:5432/development` | Postgres database URL of database storing the resource data; For Helm, these components are provided individually.                                                                                         |
 | `ENABLE_AUTO_PEERING`                                 |                                                                                                                                         | `false`                                                     | Flag to enable auto peering. View [documentation](/concepts/interledger-protocol/peering#auto-peering).                                                                                                    |
+| `ENABLE_SPSP`                                         |                                                                                                                                         | `true`                                                      | enables SPSP route                                                                                                                                                                                         |
```

Comment:
should we link to SPSP glossary item here?

Reply:
Sure. I didn't really know what to put there. Was hoping the writers had a good idea ;-)